frontend/src/constants/message.ts

```diff
@@ -76,6 +82,9 @@ const CLIENT_ERROR_MESSAGE = {
     FAILURE_EXIT_GROUP: '모임 참여 취소를 실패했어요.',
     FAILURE_EDIT_GROUP: '모임 정보 수정을 실패했어요.',
     FAILURE_LIKE_GROUP: '모임 찜하기를 실패했어요.',
+    NO_THUMBNAIL: '추가된 썸네일이 없어요. 먼저 사진을 추가해주세요.',
+    FAILURE_EDIT_THUMBNAIL: '썸네일 수정을 실패했어요.',
+    NOT_ALLOWED_THUMBNAIL_TYPE: '파일 형식은 jpg 또는 png여야 합니다.',
```

Comment:
여기도 다른 알림 메시지처럼 `~해요` 체를 사용하면 어떨까요?

```suggestion
    NOT_ALLOWED_THUMBNAIL_TYPE: '파일 형식은 jpg 또는 png여야 해요.',
```

Reply:
오 그 부분을 고려를 못했네요;
알려주셔서 감사합니다! 바로 수정할게유